tools/read_netcdf/transpose.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+gawk '
```

Comment:
can this tool be used? https://toolshed.g2.bx.psu.edu/view/iuc/datamash_transpose/1cb70a56fc89

Reply:
or the underlying datamash tool?

Reply:
Thank for your precious comments ! I had in mind the conda packaging work, and i will teach myself how to do it asap

Reply:
Let me know if you need any help!

Reply:
Indeed datamash worked pretty well ! good call & thx again

Reply:
Cool!